src/main/java/easy/gc_coffee_api/usecase/file/strategy/LocalFileStorageStrategy.java

```diff
@@ -0,0 +1,31 @@
+package easy.gc_coffee_api.usecase.file.strategy;
+
+import easy.gc_coffee_api.usecase.file.vo.FileUrl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+@Qualifier("localStorage")
```

Comment:
다형성 사용하셔서 잘 구현 하셨는데 
Qualifier는 주입 받을 때 사용하는 거라 지금은 사용 하실 필요가 없는 것 같습니다!

빈 이름을 따로 지정하시고 싶으면 아래와 같이 하시면 될 것 같아요!
```suggestion
@Component(value = "localStorage")
```